src/main/java/qna/domain/BaseEntity.java

```diff
@@ -0,0 +1,30 @@
+package qna.domain;
+
+import java.time.LocalDateTime;
+
+import javax.persistence.Column;
+import javax.persistence.EntityListeners;
+import javax.persistence.MappedSuperclass;
+
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+@MappedSuperclass
+@EntityListeners(AuditingEntityListener.class)
+public class BaseEntity {
+
+	@Column(nullable = false)
+	@CreatedDate
+	private LocalDateTime createAt;
+	@LastModifiedDate
+	private LocalDateTime updateAt;
```

Comment:
`BaseEntity`로 분리 잘해주셨네요 👍  

되도록 개행으로 분리하여 가독성을 높여주시면 좋겠어요. 
전체적으로 안지켜진 부분이 있는데 적용 부탁드립니다 🙏 

```suggestion
@MappedSuperclass
@EntityListeners(AuditingEntityListener.class)
public class BaseEntity {

	@Column(nullable = false)
	@CreatedDate
	private LocalDateTime createAt;
	
	@LastModifiedDate
	private LocalDateTime updateAt;
```

Reply:
네 👍  해당 코드는 개행이 포함된 형태로 수정하겠습니다. 리뷰감사합니다.